Tests/MongoSwiftTests/TestUtils.swift

```diff
@@ -37,10 +38,21 @@ class MongoSwiftTestCase: XCTestCase {
     /// If the environment variable does not exist, this will use a default of "mongodb://127.0.0.1/".
     static var connStr: String {
         if let connStr = ProcessInfo.processInfo.environment["MONGODB_URI"] {
+            if topologyType == .sharded {
+                let uri = mongoc_uri_new(connStr)
+                let hosts = mongoc_uri_get_hosts(uri)!
```

Comment:
>  Returns NULL if this URI’s scheme is “mongodb+srv://”.

do we care about checking for that case?

Reply:
added a guard for this